rpc/executables.go

```diff
@@ -4,18 +4,20 @@ import (
 	"context"
 
 	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/starknet.go/contracts"
 )
 
-// Get the contract class definition in the given block associated with the given hash
+// Get the CASM code resulting from compiling a given class
```

Comment:
In this method, is it just renaming, or are there any actual changes in behavior?